src/lib/error.rs

```diff
@@ -40,6 +49,24 @@ impl std::convert::From<rtnetlink::Error> for NisporError {
     }
 }
 
+impl std::convert::From<std::ffi::FromBytesWithNulError> for NisporError {
+    fn from(e: std::ffi::FromBytesWithNulError) -> Self {
+        NisporError {
+            kind: ErrorKind::NisporBug,
+            msg: e.to_string(),
+        }
+    }
+}
+
+impl std::convert::From<std::str::Utf8Error> for NisporError {
+    fn from(e: std::str::Utf8Error) -> Self {
+        NisporError {
+            kind: ErrorKind::NisporBug,
+            msg: e.to_string(),
```

Comment:
How about `format!("Utf8Error: {}", e)` so that we could know the source this error.

Reply:
Done, thanks!